source/features/show-whitespace.css

```
@@ -1,3 +1,33 @@
+/*
+Can't use this because attributes are strings and calc() won't cast them
+ttps://stackoverflow.com/a/40815884
+
+[data-tab-size] {
+	--tab-size: attr(data-tab-size);
+}
```

Demo suggestion, do not resolve

```suggestion
[data-tab-size] {

	--tab-size: attr(data-tab-size);
}
```